bento_drop_box_service/config.py

```diff
@@ -21,6 +22,14 @@ class Config(BentoFastAPIBaseConfig):
     service_data_source: Literal["local"] = "local"
     traversal_limit: int = 16
 
+    use_https: bool = True
+    check_ssl_certificate: bool = False
+    service_s3_access_key: str = os.environ.setdefault("S3_ACCESS_KEY", "")
+    service_s3_secret_key: str = os.environ.setdefault("S3_SECRET_KEY", "")
+    service_s3_endpoint: str = os.environ.setdefault("S3_ENDPOINT", "")
+    service_s3_bucket: str = os.environ.setdefault("S3_BUCKET", "")
```

Comment:
for these variables, I'd say remove the `service_` prefix - they're used in above config because they end up being used in the `/service-info` endpoint; these aren't so we can keep it simpler